crates/core/src/config/mod.rs

```diff
@@ -96,7 +96,7 @@ impl Default for ConfigArgs {
                 ignore_protocol_checking: false,
                 gateways: None,
                 location: None,
-                bandwidth_limit: Some(12_500_000), // 10 Mbps = 1.25 MB/s * 10s window
+                bandwidth_limit: None,
```

Comment:
TODO: add performance testing to see if this feature is working as intended or not in the future